src/tests/Microsoft.PowerFx.Connectors.Tests/OpenApiParserTests.cs

```diff
@@ -209,8 +210,6 @@ public async Task ACSL_InvokeFunction()
  [content-header] Content-Type: application/json; charset=utf-8
  [body] {{""kind"":""Conversation"",""analysisInput"":{{""conversationItem"":{{""id"":""0"",""participantId"":""0"",""language"":""en-us"",""modality"":""text"",""text"":""Book me a flight for Munich""}}}},""parameters"":{{""projectName"":""project1"",""deploymentName"":""deploy1"",""verbose"":true,""stringIndexType"":""TextElement_V8""}}}}
 ";
-
-            Assert.Equal(expectedInput, input);
```

Comment:
> Assert.Equal(expectedInput, input); [](http://example.com/codeflow?start=11&length=36)

reenable

Reply:
Yes, this has been reenabled with McCall's change to normalize the newlines